cactus/page.py

```diff
@@ -75,16 +75,11 @@ def context(self, data=None, extra=None):
         """
```

Comment:
I think we should refactor this method. `data` is passed, but we never use it. `extra` is never passed. How about we remove those altogether?

Cheers,